checks/raw/permissions.go

```diff
@@ -473,7 +473,7 @@ func isReleasingWorkflow(workflow *actionlint.Workflow, fp string, pdata *permis
 			// Commonly JavaScript packages, but supports multiple ecosystems
 			Steps: []*fileparser.JobMatcherStep{
 				{
-					Run: "npx.*semantic-release",
+					Run: "(npx|pnpm|yarn).*semantic-release",
```

Comment:
Can you add a test?

I think this would be the easiest one to modify
https://github.com/ossf/scorecard/blob/6aa3bcc7f50e7c0efc293321284db15b1689b1a7/checks/permissions_test.go#L294-L303